File: app/src/main/java/org/lemonadestand/btb/features/login/models/CreatedAt.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

public class CreatedAt {
    @SerializedName("date")
    String date;

    @SerializedName("timezone_type")
    int timezoneType;

    @SerializedName("timezone")
    String timezone;


    public void setDate(String date) {
        this.date = date;
    }
    public String getDate() {
        return date;
    }

    public void setTimezoneType(int timezoneType) {
        this.timezoneType = timezoneType;
    }
    public int getTimezoneType() {
        return timezoneType;
    }

    public void setTimezone(String timezone) {
        this.timezone = timezone;
    }
    public String getTimezone() {
        return timezone;
    }

}

File: app/src/main/java/org/lemonadestand/btb/models/CreatedAt.kt
package org.lemonadestand.btb.models

import com.google.gson.annotations.SerializedName

class CreatedAt {
	@SerializedName("date")
	var date: String? = null

	@SerializedName("timezone_type")
	var timezoneType: Int = 0

	@SerializedName("timezone")
	var timezone: String? = null
}
